packages/react/index.classic.fb.js

```diff
@@ -18,7 +18,6 @@ export {
   StrictMode,
   Suspense,
   SuspenseList,
-  SuspenseList as unstable_SuspenseList, // TODO: Remove once call sights updated to SuspenseList
```

Comment:
This one technically isn't stable yet, despite being used at Meta ("Stage 2.5"). If anything I'd remove the unprefixed one and keep the unstable one. You can move the alias to the wrapper module in Meta's repo.

Reply:
should the main React package be exporting this as `unstable_SuspenseList` as well then? 

https://github.com/facebook/react/blob/main/packages/react/src/React.js#L133

Reply:
i'll follow up with a separate PR for `SuspenseList` changes – if i remove just the `SuspenseList` export here without other changes we get CI failures